### ISSUE TITLE
Question regarding `arrBuy` & `arrSell`

### ISSUE THREAD

**Issue by S1SYPHOS** [igoramadas/pandagainz#8]
Hey there,
it seems to me that both variables are unused throughout your application, which makes me wonder what they are for - they are defined [here](https://github.com/igoramadas/pandagainz/blob/master/src/bitpanda.ts#L323) and [here](https://github.com/igoramadas/pandagainz/blob/master/src/bitpanda.ts#L336) for reference.

Cheers,
S1SYPHOS

**// Edit:**
Come to think about it, I don't get [this line](https://github.com/igoramadas/pandagainz/blob/master/src/bitpanda.ts#L263) either, since it's the end of the loop anyway .. but I'm not great with JavaScript, I've got to admit .. thanks for any pointer in the right direction!

**Comment by igoramadas** [comment]
Well... I won't lie that the code is kind-of-unfinished. The reason the arrBuy and arrSell exist is because I had plans to do some extra calculations and logging for the qualified trades. So I would use both arrays to iterate them. But haven't done it afterwards, so essentially they are useless the moment.

The check for the trade type on line 263 is trash reminiscent of a previous reporting code, it can be deleted as well:
https://github.com/igoramadas/pandagainz/blame/6332545ddafa97c6b19034ec645423030e79f394/src/bitpanda.ts#L208

Once I have some free time from work later this month I'll hopefully revisit this project and do some refactoring to get rid of the unnecessary stuff.

TL;DR; - you are right in both cases, code not needed.

**Comment by S1SYPHOS** [comment]
Thanks for letting me know - I already finished my Python port of `bitpanda.ts`, I'll upload it later on and go from there, so thanks again for this library!

**Comment by S1SYPHOS** [comment]
Hey again,
I published my Python port of your package [here](https://github.com/S1SYPHOS/bitpynda/blob/main/src/api/bitpanda.py) as part of a CLI utility for generating reports on Bitpanda portfolios, either via API or from exported `.csv` trading files.

Thanks for your great work, it's really appreciated!

**Comment by igoramadas** [comment]
Added a link on the README 🙂
